src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 100;
+    private static final int FIGURES_COUNT = 5;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private static final int DEFAULT_RADIUS = 10;
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURES_COUNT);
+        switch (figureNumber) {
+            case 0:
+                return createCircle();
+            case 1:
+                return createSquare();
+            case 2:
+                return createRectangle();
+            case 3:
+                return createRightTriangle();
+            case 4:
+                return createIsoscelesTrapezoid();
+            default:
+                return getDefaultFigure();
+        }
+    }

```

Comment:
You can pass random values to the constructor of a figure instead of generating them inside figure classes. It's better to generate all random values in this 'FigureSupplier' class.